_includes/libraries-from-samples.md

```diff
@@ -23,33 +23,40 @@ In this section, you will use the samples created already to create libraries.
 1. A table will appear. Enter the library information:
     * _Library Name_: Leave blank as this will be filled in automatically after save.
     * _Library Alias_: Leave blank as this will be filled in automatically.
+{% if include.detailed == true %}
+    * _Description_: `Library ((Tissue type)(individual))`, e.g. `Library P1`
+{% else %}
+    * _Description_: `Library (row number)`, e.g. `Library 1`
+{% endif %}
     * _Matrix Barcode_: As before, usually this would be scanned by the hand scanner. In this tutorial, enter matrix
       barcodes in the form `(project {% if include.detailed %}short{% endif %} name)_(row number)_lib`, e.g.
       `PROJ_1_lib`.
+    * _SOP_: Choose any value if required.
+    * _Thermal Cycler_: Choose any value if required.
 {% if include.detailed == true %}
-    * _Description_: `Library ((Tissue type)(individual))`, e.g. `Library P1`
     * _Design_: Select any
-{% else %}
-    * _Description_: `Library (row number)`, e.g. `Library 1`
 {% endif %}
-    * _Platform_: {{ site.platform_type }}
-    * _Type_: {{ site.library_type }}
+    * _Platform_: `{{ site.platform_type }}`
+    * _Type_: `{{ site.library_type }}`
 {% if include.detailed == false %}
     * _Selection_: Select any
     * _Strategy_: Select any
 {% endif %}
-    * _Index Kit_: {{ site.index_kit }}
-    * _Index 1_ and _Index 2_ (if available): Select any combination of indices you wish.
-      Select different indices for each library. Selecting the same indices for two
-      different libraries would make it unwise to pool those two libraries
-      together later.
+    * _Index Kit_: `{{ site.index_kit }}`
+    * _Index 1_ and _Index 2_ (if available): Select any combination of indices
+      you wish. Select different indices for each library. Selecting the same
+      indices for two different libraries would make it unwise to pool those two
```

Comment:
Remind me - does MISO prevent the user from creating pools with colliding indices?

Reply:
It may - depends on configuration